mentors/cpp/v.json

```
@@ -1 +1,10 @@
-[]
+[
+	{
+	"github_username": "Vaibhavi24",
+    "name": "Vaibhavi Singh",
+    "link_text": "Linkedin profile",
+    "link_url": "https://www.linkedin.com/in/vaibhavi-singh-45619a138/",
+    "avatar_url": null,
+    "bio": "I've written code in C++ for more than 6 yrs. Mentored in Open Source programs, love to understand and simplify abstract concepts."
+    }
```

```suggestion
   {
      "github_username": "Vaibhavi24",
      "name": "Vaibhavi Singh",
      "link_text": "Linkedin profile",
      "link_url": "https://www.linkedin.com/in/vaibhavi-singh-45619a138/",
      "avatar_url": null,
      "bio": "I've written code in C++ for more than 6 yrs. Mentored in Open Source programs, love to understand and simplify abstract concepts."
   }
```

Just some minor indentation corrections to make.

---

Thanks @rpottsoh! Made the changes! Pls review :)